project/Dependencies.scala

```diff
@@ -318,10 +318,10 @@ object Dependencies {
         // jackson-module-scala is only available for Scala 3 from 2.13.0 onwards.
         // since we don't depend on it ourselves, but provide it as a transitive
         // dependency for convenience, we can leave it out for Scala 3 for now,
-        // and depend on 2.13.0-rc1 for our tests. Eventually we should consider
+        // and depend on 2.13.2 for our tests. Eventually we should consider
         // whether to update all jackson artifacts for Scala 3.
         if (scalaVersion.value.startsWith("3."))
-          Seq("com.fasterxml.jackson.module" %% "jackson-module-scala" % "2.13.0-rc1" % Test)
+          Seq("com.fasterxml.jackson.module" %% "jackson-module-scala" % "2.13.2" % Test)
```

Comment:
> I'm not sure why jackson-module-scala is not needed for runtime - since it is referenced in https://github.com/akka/akka/blob/main/akka-serialization-jackson/src/main/resources/reference.conf#L20

@raboof do you know why it was added only as test dependency?

Reply:
There has been some discussion on https://github.com/akka/akka/issues/31237. I would favour the idea of having the scala 3 version (and that version only) depend on jackson-module-scala 2.13.x (including other transitive jackson dependencies).

Reply:
> @raboof do you know why it was added only as test dependency?

I think it seemed 'wrong' to me to have public transitive dependencies on conflicting versions of Jackson (and an '-rc' version at that), but I didn't dare update all of Jackson for Scala 3 yet, so I took the 'conservative' option of only adding it as a test dependency (and adding that comment above it).

Reply:
https://github.com/akka/akka/blob/main/akka-serialization-jackson/src/main/resources/reference.conf#L20 will only work if jackson-module-scala is on the runtime classpath.